features/users/mute.feature

```diff
@@ -0,0 +1,152 @@
+Feature: Muting
```

Comment:
I just noticed the blocking feature is named blocking -- can you change this to muting to match?